charts/chronicle/templates/tests/api-test.yaml

```diff
@@ -16,28 +16,62 @@ spec:
       serviceAccountName:  {{ include "lib.serviceAccountName" . }}
       automountServiceAccountToken: true
       initContainers:
-      {{- if .Values.auth.required }}
         - name: wait-for-id-provider
           {{- include "lib.image" (dict "imageRoot" .Values.test.api.image "global" .Values.global ) | nindent 10 }}
-          command: [ "sh", "-ec" ]
+          command: [ "sh", "-c" ]
           args:
             - |
-              URL={{ include "chronicle.jwksUrl" . }}
-              scheme=$(echo $URL | cut -f 1 -d :)
-              hostAndPort=$(echo $URL | cut -f 3 -d /)
-              HOST=$(echo $hostAndPort | cut -f 1 -d :)
-              port=$(echo $hostAndPort | awk -F: '{print $2}')
-              if [ "$scheme" = "http" ]; then
-                defaultPort=80
-              elif [ "$scheme" = "https" ]; then
-                defaultPort=443
+              {{- if or (.Values.auth.jwks.url) (.Values.devIdProvider.enabled) }}
```

Comment:
now you are doing this even in the case where auth.required==fale, and devIdProvider.enabled==false, in which case all of the following is going to go wrong.